etc/config.xml

```diff
@@ -14,6 +14,7 @@
                 <validation_strategy>validation-stop-on-errors</validation_strategy>
                 <allowed_error_count>0</allowed_error_count>
                 <ignore_duplicates>0</ignore_duplicates>
+                <category_path_seperator>/</category_path_seperator>
```

Comment:
I do not think we should do this, because this is a backwards-incompatible change. If people did not set this config value in their current systems, this will change it and possibly breaks imports.

Reply:
Without a default, and without it being set in the config, you can't import. I think I had assumed the separator would be /. 

I think the error getting thrown was 

> In Import.php line 273:
> 
>   Please enter a correct entity model.

And I had to work backwards from that, and ended up seeing split getting called with no separator. I guess we could call it a documentation error?

I don't think adding the default will break anything though, as it wasn't working before, and anyone who sets a value will be ignoring the default.

Reply:
Pretty sure it only breaks with PHP 8. With PHP 7, it would silently be ignored. And if we now configure it with a slash, people importing a category called "Shirts / T-Shirts" would end up with a "Shirts" parent category and a "T-Shirts" child category, whereas previously, they just got one category called "Shirts / T-Shirts". So I still think this needs to be reverted in order to not break existing imports :-/

Reply:
No problem. I added a revert for that commit